frontend/src/__tests__/cypress/cypress/pages/modelRegistry/modelVersionDetails.ts

```diff
@@ -32,6 +32,23 @@ class ModelVersionDetails {
     return cy.findByTestId('pipeline-run-link');
   }
 
+  shouldNotHavePipelineRunLink() {
+    this.findPipelineRunLink().find('a').should('not.exist');
+    return this;
+  }
+
+  findInfoPopover() {
+    return cy.findByTestId('project-info');
+  }
+
+  shouldHaveProjectInfoText() {
```

Comment:
If this is specifically checking for Admin access warning, the function name can be 

```suggestion
  shouldHaveProjectAccessWarning() {
```

Reply:
I’d say it’s more of an info message than a warning
changing it to `shouldHaveProjectAccessInfo()` make more sense??

Reply:
yes, better